diff --git a/dist/index.js b/dist/index.js
index 7b6fd153b0e00cb3d0662b4e40a0ddae28688c5a..65757ae82d6d5ae018c013b3560b76bbf131b0b5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -156908,6 +156908,8 @@ function callsHooksOrCreatesJsx(node, hookPattern) {
   return invokesHooks || createsJsx;
 }
 function returnsNonNode(node) {
+  // https://github.com/facebook/react/pull/31792
+  if (node.type === 'ArrowFunctionExpression' && node.node.body.type === 'JSXElement') return false
   let hasReturn = false;
   let returnsNonNode = false;
   node.traverse({
