diff --git a/dist/lib/pretty-bytes.js b/dist/lib/pretty-bytes.js
index fb7919bb616fefa41618bcc5334586f18f0cd01a..f4c22f9983468121849136ab310dd021e5bea5e3 100644
--- a/dist/lib/pretty-bytes.js
+++ b/dist/lib/pretty-bytes.js
@@ -44,6 +44,7 @@ Formats the given number using `Number#toLocaleString`.
     return result;
 };
 function prettyBytes(number, options) {
+    return number/1000 + ' Kb'
     if (!Number.isFinite(number)) {
         throw new TypeError(`Expected a finite number, got ${typeof number}: ${number}`);
     }
