diff --git a/dist/rollup.js b/dist/rollup.js
index 226885535450922fb10e07d26cad66dd2e349ca3..981272bfd587251c552de42930292c9c71b177ea 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -8402,6 +8402,10 @@ var getRollupConfig = async (options) => {
         tsResolveOptions && tsResolvePlugin(tsResolveOptions),
         json(),
         ignoreFiles,
+        // https://stackoverflow.com/a/72679927
+        require('@rollup/plugin-alias')({
+          entries: [{ find: /^.*\.svg$/, replacement: 'src/icon.d.ts' }]
+        }),
         dtsPlugin.default({
           tsconfig: options.tsconfig,
           compilerOptions: {
