diff --git a/dist/rollup.js b/dist/rollup.js
index 0f6400eedfad49091ca952ee5863bd027e3b8417..06bb0f91a7b6f4cf526d36d194f45d084afa7ad3 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -12,6 +12,7 @@ var _chunkGEVDEBBWjs = require('./chunk-GEVDEBBW.js');
 
 
 
+const alias = require('@rollup/plugin-alias');
 
 
 
@@ -7929,6 +7930,10 @@ var getRollupConfig = async (options) => {
         (0, import_rollup_plugin_hashbang.default)(),
         json(),
         ignoreFiles,
+        // https://stackoverflow.com/a/72679927
+        alias({
+          entries: [{ find: /^.*\.svg$/, replacement: 'src/icon.d.ts' }]
+        }),
         dtsPlugin.default({
           tsconfig: options.tsconfig,
           compilerOptions: {
